packages/js/components/src/collapsible-content/collapsible-content.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { createElement, useState } from '@wordpress/element';
+import { Icon, chevronDown, chevronUp } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+
+export type CollapsedProps = {
+	initialCollapsed?: boolean;
```

Comment:
Updated this to `initialCollapsed` instead of `isCollapsed` since it does not control state after initialization.